### PR TITLE
Add a basic example which blinks an LED

### PR DESCRIPTION
**Pull request by jessebraham**
Pretty self-explanatory. If there are any issues please let me know and I will get them fixed up.

### CODE REVIEW

File: examples/blinky.rs
Comment (jeandudey) [inline examples/blinky.rs:13]: I don't intend to change this but to keep this as a side note, in the PR #7 this can be calculated at run time depending on the clock source used.

Comment (jessebraham) [reply examples/blinky.rs:13]: Thanks for pointing that out. I'm fine with waiting until that PR has merged and updating the example if that's preferable.

File: examples/blinky.rs
Comment (MabezDev) [inline examples/blinky.rs]: xtensa-lx6-rt now has this function: https://github.com/esp-rs/xtensa-lx6-rt/blob/master/src/lib.rs#L39, lets import that instead.

File: .cargo/config
Comment (MabezDev) [inline .cargo/config]: This is no longer required, the new llvm fork supports debug info.

File: Cargo.toml
Comment (MabezDev) [inline Cargo.toml]:
```suggestion
[profile.dev]
incremental = false
```

File: Cargo.toml
Comment (MabezDev) [inline Cargo.toml]:
```suggestion
[profile.release]
lto = true
incremental = false
codegen-units = 1
```

**Review by MabezDev — CHANGES_REQUESTED** [review]
Looking good! Sorry I didn't get to this sooner. Just a few bits to change:)

File: examples/blinky.rs
Comment (jeandudey) [reply examples/blinky.rs:13]: Maybe merge this PR first because #7 has been superseded by #11

File: .cargo/config
Comment (jessebraham) [reply .cargo/config]: Just for clarification, am I removing the entire `rustflags` array, its first 2 elements, or just the `debuginfo` line?

**Comment by jessebraham** [comment]
I think everything should be okay now. If I still don't have the `config` file correct just let me know and I'll make the necessary adjustments.

**Comment by MabezDev** [comment]
Perfect! Thank you!

File: .cargo/config
Comment (arjanmels) [reply .cargo/config]: @MabezDev Which exact llvm fork are you using? If I use https://github.com/espressif/llvm-project/tree/xtensa_release_9.0.1 I still get CFI not supported errors.

Comment (MabezDev) [reply .cargo/config]: @arjanmels I'm using that fork; it should work. Note that if you're trying to compile .S assembly files, it will be passed to the gcc assembler at the moment, which will produce these errors. Inline assembly works though.

Comment (arjanmels) [reply .cargo/config]: Strange, not working for me. I recompiled llvm and rust, but no luck, must be missing something.

This is what I get if I check my version (afraid it may not be super helpfull:

$RUSTC --version --verbose
rustc 1.44.0-dev
binary: rustc
commit-hash: unknown
commit-date: unknown
host: x86_64-unknown-linux-gnu
release: 1.44.0-dev
LLVM version: 9.0

Comment (MabezDev) [reply .cargo/config]: That is weird, so you can't create debug info at all? I've defintely got it working on my machine: https://github.com/espressif/llvm-project/issues/10#issuecomment-575854084

Ive been pretty tied up this week, but today I've been revisiting the rust fork, looking to improve the ABI calls, and use the built in llvm module, which *might* fix your issue.

Comment (arjanmels) [reply .cargo/config]: No debug info not working for me. If I understand your comment for https://github.com/espressif/llvm-project/issues/10#issuecomment-575854084 correct you are working with a local rebase, so theoretically could be something different?

More likely that I did something wrong perhaps, but I don't know what. Anyway I am managing ok without the debugger at the moment as I have the serial output up and running, but debugger would be nice of course.

Comment (MabezDev) [reply .cargo/config]: That rebase was temporary, the current default branch of the fork has the correct changes in it.

**Pull request merged**
